lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
You are right ... when the other person still shares with you, the contact doesn't get deleted ... So the data generator probably only creates one-way contacts? because user.contacts were empty after reload.